content/en/users/tutorials/htc-job-submission/_index.md

```diff
@@ -0,0 +1,397 @@
+---
```

Comment:
as the list of tutorials is growing, maybe we could start grouping therm by area? like data, compute, etc?

Reply:
Yes, it could make sense, I wasn't sure how to order them. But it should be discussed and this is for a later/separate PR.
We should also ensure that we properly link them from where users can found them, on the various service specific pages.

Reply:
Closing, to be discussed elsewhere/at a later stage.